signing_service/signing_service/signing_service.py

```diff
@@ -31,6 +31,7 @@ class SigningService:
         'current_reconnect_delay',
         'socket',
         'was_sigterm_caught',
+        'concent_public_key',
```

Comment:
Shouldn't this be in different commit? In this commit you only changing layout. This should be moved to: `Add concent_public_key argument, its validation and tests`

Reply:
:+1: done